Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
Just a heads up there is a lighter version of this that Rails provides which has a couple of benefits:
- don't need a third party gem
- don't need to modify query code, i.e. `find`

Example:

```ruby
class Event
   to_param :title
end
```

I haven't used this with Globalize before but assume it should work.